frontend/src/metabase/visualizations/components/TableInteractive.jsx

```diff
@@ -281,7 +282,7 @@ export default class TableInteractive extends Component<*, Props, State> {
                     })}
                     onClick={isFilterable && this.showPopover.bind(this, rowIndex, columnIndex)}
                 >
-                    <Value value={cellData} column={column} onResize={this.onCellResize.bind(this, columnIndex)} />
+                    <Ellipsified><Value value={cellData} column={column} onResize={this.onCellResize.bind(this, columnIndex)} /></Ellipsified>
```

Comment:
This line is getting long, could you put `<Value>` on it's own line (and indent like the rest)